### PR TITLE
adding section 1 to the html and css files

### PR DESCRIPTION
**Pull request by Moamin-AbuEwaida**
I added section 1 of the latest wire frame which contains (our program) part of the website including the button

### CODE REVIEW

**Comment by Moamin-AbuEwaida** [comment]
I've just pushed another commit as I replaced the button tag into a link tag and added the page where it suppose to go to when click in it

File: index.html [hunk 1]
```diff
@@ -21,6 +21,19 @@
...
+    <section class="section 1">
+      <div class="section1 text">
+        <h2>Our Program</h2>
+        <p> Our <span class="colored"> 8-month </span> program is divided into <span class="colored">12 modules </span>. The last module is
+          a <span class="colored">6-weeks intensive Final Project </span> where students work on a real project 
+          with a real partner. Our training takes place at <span class="colored">BeCentral </span> (Brussels)  
+          every <span class="colored">Sunday </span>. During the week, students are supported online in 
+          their <span class="colored">self-study </span>, homework and peer learning. </p>
+      </div>
+      <a class="section1Btn" href ="/program.html">discover more &#8594; </a>
```
Comment (danielhalasz) [inline index.html:34]: GitHub Pages is very picky with links, and although normally it is correct in a browser, I think it only works correctly without using the / character, at least before folders..so for example /img/logo.svg does not work, but img/logo.svg does 😄

Comment (Moamin-AbuEwaida) [reply index.html]: last commit is on adding section 2 image aside to section 1 contect

Comment (Moamin-AbuEwaida) [reply index.html]: > GitHub Pages is very picky with links, and although normally it is correct in a browser, I think it only works correctly without using the / character, at least before folders..so for example /img/logo.svg does not work, but img/logo.svg does smile

yeah may be, i didn't pay attention to github links thing specially with adding the '/' to the links or images :grin:

**Comment by danielhalasz** [comment]
resolved the conflicts

**Comment by Moamin-AbuEwaida** [comment]
> resolved the conflicts

I was trying to resolve the conflict myself but got a little bit lost :thinking: 
could u please share with me the steps u took on slack :grinning:

**Review by danielhalasz — APPROVED** [review]
YESS

**Review by bringmetheaxx — APPROVED** [review]
good!

**Comment by Moamin-AbuEwaida** [comment]
i'll give it a shot to change the not working style

**Pull request merged**